setup.cfg

```diff
@@ -55,6 +55,93 @@ test =
     hypothesis>=5.5.3
     pytest>=6.0
     pytest-xdist>=1.31
+# optional extras for recommended dependencies
+# see: doc/source/getting_started/install.rst
+performance =
+    bottleneck>=1.3.2
+    numba>=0.53.0
+    numexpr>=2.7.1
+computation =
+    numba>=0.53.0
+    scipy >=1.7.1
+    xarray>=0.19.0
+aws =
```

Comment:
I think `fsspec` should be included in `aws` and `gcp`

Reply:
This one of the advantages of the `extras_require` system: we can and should trust packages to manage their own required dependencies.

The `s3fs` package enforces `fsspec` (https://github.com/fsspec/s3fs/blob/main/requirements.txt), ditto `gcsfs` (https://github.com/fsspec/gcsfs/blob/main/requirements.txt) and therefore Pandas should not pin `fsspec`.  It just creates a way for us to break the dependencies tree.

Unless I'm missing a way in which pandas, for aws and gcp, needs `fsspec` functionality independent of  `s3fs`|`gcsfs` sufficient to risk divergent `fsspec` versions from those packages?